app/scripts/controllers/preferences-controller.ts

```diff
@@ -140,6 +140,8 @@ export type PreferencesControllerState = Omit<
   useRequestQueue: boolean;
   ///: BEGIN:ONLY_INCLUDE_IF(build-flask)
   watchEthereumAccountEnabled: boolean;
+  ///: END:ONLY_INCLUDE_IF
+  ///: BEGIN:ONLY_INCLUDE_IF(solana-snap)
```

Comment:
is this one properly closed?

Reply:
it is, it closes the above open fence in L141. Also the builds would fail if one code fence was misplaced.